Makefile

```diff
@@ -32,9 +32,8 @@ release:
 	for env in internal-dev-sandbox internal-qa-sandbox sandbox; do \
 		cat ecs-proxies-deploy.yml | sed -e 's/{{ SPINE_ENV }}/veit07/g' | sed -e 's/{{ SANDBOX_MODE_ENABLED }}/1/g' > dist/ecs-deploy-$$env.yml; \
```

Comment:
I know internal-qa-sandbox doesn't actually connect to Spine, but maybe make the same change here just for consistency?

Reply:
Actually never mind, we don't do it for sandbox either